app/static/src/app/userMessages.ts

```diff
@@ -24,9 +24,16 @@ export default {
         columnToFitPrerequisites: "Please link at least one column in order to set target to fit."
     },
     run: {
-        compileRequired: "Model code has been updated. Compile code and Run Model to view updated graph.",
-        runRequired: "Model code has been recompiled or options have been updated. Run Model to view updated graph.",
-        notRunYet: "Model has not been run."
+        compileRequired: "Model code has been updated. Compile code and Run Model to update.",
+        notRunYet: "Model has not been run.",
+        updateReasons: {
+            prefix: "Plot is out of date:",
```

Comment:
Could these repeated messages be referenced from a single place?

Reply:
all the strings are _just_ different enough to make sharing them a bit tedious unfortunately. Once we come to internationalising this though we'll want to do a big run through and filter out the redundancy